docs/assignment3.md

```diff
@@ -1 +1,306 @@
 # Assignment 3
+
+Maximum number of words for this document: 18000
+
+**IMPORTANT**: In this assignment you will fully model and impement your system. The idea is that you improve your UML models and Java implementation by (i) applying (a subset of) the studied design patterns and (ii) adding any relevant implementation-specific details (e.g., classes with “technical purposes” which are not part of the domain of the system). The goal here is to improve the system in terms of maintainability, readability, evolvability, etc.    
+
+**Format**: establish formatting conventions when describing your models in this document. For example, you style the name of each class in bold, whereas the attributes, operations, and associations as underlined text, objects are in italic, etc.
+
+### Summary of changes of Assignment 2
+Author(s): `Torstein Thomassen`
+
+Bullet list summarizing all the changes from Assignment 2 feedback:
```

Comment:
would have been nice to add a bit more detail to these bullet points. As it stands, these are not bullet changes but rather vague points that insinuate changes